src/screens/SSIContactAddScreen/index.tsx

```diff
@@ -110,7 +94,7 @@ class SSIContactAddScreen extends PureComponent<IProps, IState> {
       filter: [
         {
           contact: {
-            displayName: name,
+            legalName: name,
```

Comment:
Is this correct?

Reply:
displayName is not unique and currently we only support organizations

Reply:
Okay suggest to add a comment about that then

Reply:
added a comment